client/src/styles/components/_quiz.scss

```diff
@@ -112,6 +112,7 @@ body.dark #timer {
 
 #next-btn {
   display: flex;
+  flex-direction: column;
```

Comment:
Why do we need this?

Reply:
To display the error message above the validation button, and not next to it.

Reply:
Okay 👍